packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -42,6 +42,8 @@ const createYargs: (argv?: string[], cwd?: string) => Argv = require('yargs/yarg
 export interface TheiaBrowserWindowOptions extends BrowserWindowConstructorOptions {
     isMaximized?: boolean;
     isFullScreen?: boolean;
+    // screenLayout record the monitor information, so we can know if the monitors have been changed.
+    screenLayout: string;
```

Comment:
```suggestion
    /**
     * Represents the complete screen layout for all available displays.
     * - Used to determine if the layout was updated since the electron window was last opened.
     */
    screenLayout: string;
```

Reply:
Thanks. have changed.